connectors/automation-anywhere/element-templates/automation-anywhere-connector.json

```diff
@@ -0,0 +1,582 @@
+          "value": "userNameApiKeyAuth"
+        },
+        {
+          "name": "Authentication (refresh) token",
+          "value": "tokenAuth"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.withoutBindingType"
+      }
+    },
+    {
+      "id": "credentialsInBodyAuthType",
+      "group": "authentication",
+      "type": "Hidden",
+      "value": "credentialsInBody",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.type"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "userNamePasswordAuth",
+          "userNameApiKeyAuth"
+        ]
+      }
+    },
+    {
+      "group": "authentication",
+      "type": "Hidden",
+      "value": "noAuth",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.authentication.type"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "userNamePasswordAuth",
+          "userNameApiKeyAuth"
+        ]
+      }
+    },
+    {
+      "id": "tokenAuthType",
+      "group": "authentication",
+      "type": "Hidden",
+      "value": "noAuth",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.type"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "tokenAuth"
+        ]
+      }
+    },
+    {
+      "description": "method for authentication",
+      "type": "Hidden",
+      "value": "post",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.method"
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "userNamePasswordAuth",
+          "userNameApiKeyAuth"
+        ]
+      }
+    },
+
+    {
+      "label": "Username",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.body.username"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "userNamePasswordAuth",
+          "userNameApiKeyAuth"
+        ]
+      }
+    },
+    {
+      "label": "Password",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.body.password"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "userNamePasswordAuth"
+      }
+    },
+    {
+      "label": "API key",
```

Comment:
is it correct that we are calling it API Key and not token here?
if this is the official name, then we don't need to change it :-) I am just curious